### ISSUE TITLE
Set an event as your favorite

### ISSUE THREAD

**Issue by kevinderudder** [jfversluis/CfpExchange#90]
It would be great if you could get notified when an event opens a CFP.

**Comment by jfversluis** [comment]
Yes! Very awesome feature. Yesterday I also thought; I would like to see if I already submitted to a CFP. So mark them as such. But for all this greatness we would need to have accounts...
